docs/_partner_api/authentication.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Authentication
+order: 4
+layout: post
+redirect_from: /partner_api/
+---
+
+# Authentication
+
+There are two ways to authenticate with the Partner API.
+
+1. Your application's `client_id` which you will receive once you are approved for access to the API
+2. A user's access token
+
+Which authentication method you should use depends on which endpoint(s) you are using. Review each endpoint's documentation to understand which parameters are required.
+
+> Note: while we do generate a `client_secret`, the type of grant we use (`implicit`) doesn't need it so it's not something we provide.
+
+## Access Token
+
+For resources that require a valid access token you can use the [OAuth2 protocol](https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2). At the moment, we only permit the [`implicit`](https://tools.ietf.org/html/rfc6749#section-4.2) grant type. Should your use case require a different grant type [send us your request](mailto:partners@zapier.com). There's also a [suggested workaround on how to work with an implicit oauth flow below](#workaround-for-implicit-only).
+
+### Procuring a Token
+Construct the following URL, and redirect the user to authorize your application:
+
+```
+https://zapier.com/oauth/authorize?client_id={client_id}&redirect_uri={redirect_uri}&scope={scope}
+```
+
+|      Parameter      | Requirement | Explanation                                                                                                                                                                                                         |
+| :-----------------: | :---------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|    **client_id**    |  Required   | Your application ID.                                                                                                                                                                                                |
+|  **redirect_uri**   |  Required   | The URI you provided in the sign-up form. If you need to modify this, you'll need to [send us a request](mailto:partners@zapier.com).                                                                               |
+|  **response_type**  |  Required   | Use `token`.                                                                                                                                                                                                        |
+|      **scope**      |  Optional   | Space (`%20`) separated variable. See each resource for their required scope, if any.                                                                                                                               |
```

Comment:
It looks like all these params should be properly querystring encoded, which is only alluded to by "space" being `%20`. I might restructure this line to be something like 

```
Space separated values. See each... This (and all other params) should be properly [url encoded](https://en.wikipedia.org/wiki/Percent-encoding).